.github/workflows/grid_client_nightly.yml

```diff
@@ -111,53 +111,33 @@ jobs:
             exit 1
           fi
 
-      - name: Test Results
+      - name: Find issues
+        if: failure()
+        id: find-issues
+        env:
+          GH_TOKEN: ${{ github.token }}
         run: |
-          echo Dynamic Single Vm: ${{ steps.dynamicsinglevm.outcome }}
-          echo Multiple Vm: ${{ steps.multiplevm.outcome }}
-          echo Kubernetes: ${{ steps.kubernetes.outcome }}
-          echo Vmq Qsfs: ${{ steps.vmqsfs.outcome }}
-          echo Kubernetes Qsfs: ${{ steps.kubernetesqsfs.outcome }}
-          echo Kvstore: ${{ steps.kvstore.outcome }}
-          echo Zdb: ${{ steps.zdb.outcome }}
-          echo Delete all contracts: ${{ steps.deleteall.outcome }}
-
-      - name: Check test script status
-        if: |
-          steps.dynamicsinglevm.outcome != 'success' ||
-          steps.multiplevm.outcome != 'success' ||
-          steps.kubernetes.outcome != 'success' ||
-          steps.vmqsfs.outcome != 'success' ||
-          steps.kubernetesqsfs.outcome != 'success' ||
-          steps.kvstore.outcome != 'success' ||
-          steps.zdb.outcome != 'success' ||
-          steps.deleteall.outcome != 'success'
-        run: exit 1
+          ISSUE_COUNT=$(gh issue list --search "${{github.workflow}} failed during schedule on ${{ env.NETWORK }}" --json title --jq '. | length')
```

Comment:
I think the `--jq` here is a flag, no need to install the package

Reply:
You're right. removed the package and tested the workflow and it could create an issue as the gh uses jq internally.